sdk/python/resources/registry/registry-create.ipynb

```diff
@@ -197,21 +199,13 @@
     "for reg in ml_client.registries.list():\n",
     "    print(reg.name)"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "5f2c0f69",
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
```

Comment:
Please keep the kernel name same as we have for the other notebooks.

Reply:
reverted